src/main/java/com/ffxivcensus/gatherer/Gatherer.java

```diff
@@ -36,27 +35,15 @@
     public void run() {
         try {
             LOG.debug("Starting evaluation of player ID: " + getPlayerId());
-            // Parse players and write them to DB
-            PlayerBean player = PlayerBuilder.getPlayer(getPlayerId(), 1);
-            // Currently ignore deleted characters (404)
-            if(!CharacterStatus.DELETED.equals(player.getCharacterStatus())) {
-                String out = getDao().saveRecord(player);
-                LOG.debug(out);
-            }
-            RESULT_LOG.info(getPlayerId() + " - " + player.getCharacterStatus().name());
-        } catch(MySQLNonTransientConnectionException failWriteEx) { // If record fails to write due to too many connections
```

Comment:
Removed this as the connection pooling and Spring handle connections, so we no-longer have connection leaks.